### PR TITLE
fix typo from beta 5 to beta 6

### PR DESCRIPTION
**Pull request by asffp**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by ajitsonlion** [comment]
I signed it!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<a name="6.0.0-beta.5"></a>
-# [6.0.0-beta.5](https://github.com/angular/angular/compare/6.0.0-beta.4...6.0.0-beta.5) (2018-02-28)
+<a name="6.0.0-beta.6"></a>
+# [6.0.0-beta.6](https://github.com/angular/angular/compare/6.0.0-beta.6...6.0.0-beta.6) (2018-02-28)
```
Comment (devoto13) [inline CHANGELOG.md]: 6.0.0-beta.6...6.0.0-beta.6 -> 6.0.0-beta.5...6.0.0-beta.6

**Comment by ajitsonlion** [comment]
Hmm, I did something wrong :(
I dunno what I should do for ci/angular: merge status and  ci/circleci: lint .

**Comment by devoto13** [comment]
`ci/angular: merge`fails because PR is not approved. You can ignored it.
Regarding lint: you should fix commit message. See point 8 in [CONTRIBUTING.md](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-a-pull-request-pr).

**Comment by ajitsonlion** [comment]
Ok, thanks a lot :)

**Comment by alexeagle** [comment]
err sorry, I already fixed it on master. Thanks for pointing out the mistake.

**Pull request closed without merging**